lib/inspec/resources/mysql_session.rb

```diff
@@ -16,19 +16,24 @@ class MysqlSession < Inspec.resource(1)
       end
     EXAMPLE
 
-    def initialize(user = nil, pass = nil, host = "localhost", port = nil, socket = nil)
+    def initialize(user = nil, pass = nil, host = "localhost", port = nil, socket = nil, sensitive: false)
       @user = user
       @pass = pass
       @host = host
       @port = port
       @socket = socket
+      @sensitive = sensitive
       init_fallback if user.nil? || pass.nil?
       skip_resource("Can't run MySQL SQL checks without authentication") if @user.nil? || @pass.nil?
     end
 
     def query(q, db = "")
       mysql_cmd = create_mysql_cmd(q, db)
-      cmd = inspec.command(mysql_cmd)
+      cmd = if @sensitive && !@pass.nil?
```

Comment:
We could consider always passing the redact_regex and not exposing an option to the user. I can't imagine a case where the user will need to see their password echoed back to them.

Reply:
@james-stocks I like that.

My only use case was just debug to see if you have the right pass (which you should, but you know...). But I guess if you actually use `-l debug` you are good to go?

Reply:
That is an interesting option @james-stocks ! 🤔  I would definitely support always using `redact_regex` in the cases you've identified @fzipi if that's something you wouldn't mind doing.

I would be very surprised if someone were to come back in the future saying "hey, why can't I see my password in the results?" 😅

Reply:
Sounds good to me!